hw/ip/prim/dv/prim_ascon/ascon_model_dpi/vendor/ascon_ascon-c.vendor.hjson

```diff
@@ -0,0 +1,28 @@
+// Copyright lowRISC contributors.
```

Comment:
Is this vendor file being added in the correct commit? Maybe it should be in the previous one?

Reply:
The previous commit 92535cb19f37f9f912baa70665e459a9612b931f is autogernated by the tool and I don't know how to add the vendor.hjson there. I could split it off into it's own commit (and squash them), but as I'm not very familiar with the vendoring tool, I just did the step-by-step instructions at https://opentitan.org/book/util/doc/vendor.html Re-reading the text, it looks like the `--commit` parameter is not for updating and/or committing, but only for updating and then committing. I'll have a second look.